index.html

```diff
@@ -1317,10 +1317,13 @@ <h2>Our Advice for Newbies!</h2>
       <a href="advice/tiffany-k-advice.html">Tiffany K.'s Advice</a><br>
       <a href="advice/shanna-walsh-advice.html" target="_blank">Shanna's Advice</a><br>
       <a href="advice/chris-sinclair-advice.html" target="_blank">Chris Sinclair's Advice</a><br>
-      <a href="advice/ash-martindale-advice.html" target="_blank">Ash's Advice</a><br>
+      <a href="advice/ash-martindale-advice.html" target="_blank">Ash's Advice</a><br>advice
+      <a href="advice/alisha-pollitt-advice.html" target="_blank">Alisha Pollitt's advice</a><br>
+
       <a href="advice/kassie-bartow-advice.html" target="_blank">Kassie's Advice 🦋</a><br>
 	    <a href="advice/darryl-j-advice.html">Darryl Jackson’s Advice</a><br>
       <a href="advice/april-jordan-advice.html" target="_blank">April's Advice</a><br>
+main
```

Comment:
Hey @codekid92 ! 

Let's remove this line here - looks like the remnants of conflict markers. ;)

Reply:
Hi Ann, 

Please let me know if I was able to make the correct change. I was not sure if I was supposed to change this in the local advice.html file, and made the changes remotely on GitHub.